src/orange/detail/BIHBuilder.hh

```diff
@@ -51,11 +51,14 @@ class BIHBuilder
     //!@{
     //! \name Type aliases
 
-    //! Input parameters
-    struct Input
+    //! Options parameters
+    struct Options
```

Comment:
if we move the struct, this'll become `using Input = inp::BihBuilder;` as is done in the other update celeritas classes.